File: 2437-maximum-number-of-groups-entering-a-competition/maximum-number-of-groups-entering-a-competition.java
class Solution {
    public int maximumGroups(int[] grades) {
        int n=grades.length;
        int count=0;
        int stue=1;

        while(n>=stue){
            n-=stue;
            count++;
            stue++;
        }
        return count;
    }
}

File: 2437-maximum-number-of-groups-entering-a-competition/maximum-number-of-groups-entering-a-competition.py
class Solution(object):
    def maximumGroups(self, grades):
        """
        :type grades: List[int]
        :rtype: int
        """
        n=len(grades)
        count=int(math.floor((-1+math.sqrt(1+8*n))/2))
        return count
        